picard/formats/asf.py

```diff
@@ -186,8 +186,11 @@ def _save(self, filename, metadata):
         log.debug("Saving file %r", filename)
         file = ASF(encode_filename(filename))
 
-        if config.setting['clear_existing_tags']:
-            file.tags.clear()
+        if config.setting["clear_existing_tags"]:
```

Comment:
The procedure calls are not exactly the same so, I'm not sure it is relevant to try to do some factoring here. I think that it should be necessary to refactor all the File subclasses to have something more consistant.